### ISSUE TITLE
[Feature Request] Support custom classAttributeName for react-css-modules

### ISSUE THREAD

**Issue by dr2009** [pucelle/vscode-css-navigation#60]
[babel-plugin-react-css-modules](https://github.com/gajus/babel-plugin-react-css-modules) use css-modules in this way

```jsx
import React from 'react';
import './table.css';

export default class Table extends React.Component {
  render () {
    return <div styleName='table' className='global-table'>
      <div styleName='row'>
        <div styleName='cell'>A0</div>
        <div styleName='cell'>B0</div>
      </div>
    </div>;
  }
}
```

so can you support custom classAttributeName for `babel-plugin-react-css-modules` to trigger the intellisense ?
```
"CSSNavigation.classAttributeName": "styleName"
```

**Comment by pucelle** [comment]
Will consider it.

**Comment by pucelle** [comment]
Will do it after completed https://github.com/pucelle/vscode-css-navigation/issues/59 and implement a new syntax parser.

**Comment by pucelle** [comment]
Just released v1.11.0, I guess it should work right now, if what I understand is correct.
Never mind to reopen if you found more problems.
